app/controllers/posts_controller.rb

```diff
@@ -144,6 +144,6 @@ def send_irkk
 
     def send_slack
       notifier = Slack::Notifier.new Rails.application.secrets.slack_url
-      notifier.post unfurl_links: true, unfurl_media: true, text: "New post published: *[#{@post.title}](#{post_url(@post)})* by #{@post.user.display_name}"
+      notifier.post unfurl_links: true, unfurl_media: true, text: "New post published: *[#{@post.title}](#{post_url(@post, locale: 'sv')})* by #{@post.user.display_name}"
```

Comment:
`@post.title` needs to be fetched in `sv` as well right? `@post.title_sv` should be enough :)